### PR TITLE
chore(translation): pt_br sphinx translation

### LINKED ISSUE #129: Translate documentation to Portuguese
We need to have the documentation in Portuguese. 
here is a guide to integrate it with sphinx:
https://www.sphinx-doc.org/en/master/usage/advanced/intl.html

### PR DESCRIPTION
**Pull request by luabida**
fix #129 

command used to build:
`make -e SPHINXOPTS="-D language='pt'" html`

### CODE REVIEW

**Comment by luabida** [comment]
@fccoelho I was able to build the docs and generate the .mo files, but they were already excluded in `.gitignore` file, should it be added in the repo?

File: .gitignore [hunk 1]
```diff
@@ -80,6 +80,7 @@ coverage.xml
 cover/
 
 # Translations
+#
 *.mo
```
Comment (luabida) [inline .gitignore:84]: here

**Comment by luabida** [comment]
I wasn't able to test the sphinx server locally, is there any extra commands to create a webserver in localhost to check the translations?

File: .gitignore (hunk 1 above)
Comment (fccoelho) [reply .gitignore:84]: Yes, just add them to the repo.

**Comment by fccoelho** [comment]
> I wasn't able to test the sphinx server locally, is there any extra commands to create a webserver in localhost to check the translations?

I normally just open the index.html from the build directory.

**Review by fccoelho — APPROVED** [review]

**Pull request merged**